apps/nextjs-website/src/_contents/appIo/apiPath.ts

```diff
@@ -2,5 +2,5 @@ import { Path } from '@/lib/types/path';
 
 export const appIoApiPath: Path = {
   name: 'API',
-  path: '/app-io/api',
```

Comment:
I don't think this is the slug we should use.
The current API (in english) has `IO API for Public Administration Services` as a title and I think we should convert it like this: `io-api-for-public-administration-services` or translate it and convert the translation to the slug.
![image](https://github.com/pagopa/developer-portal/assets/39835990/eed88444-6b22-4a79-974e-49b027d7654f)

We should ask to the team

Reply:
`main` for all api exept for those of pagoPA